.github/workflows/Tests.yml

```diff
@@ -14,6 +14,10 @@
 
 name: Regression testing
 
+concurrency:
```

Comment:
Could you add a comment about why we want to limit concurrency?

Reply:
@umarcor added this. Not sure why. Could you please approve this?

Reply:
Added a reference to the docs about the concurrency feature  in GitHub Actions: https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pr/pull/18/commits/8d7f2e455e8cf10fcb71ac6501ed5e72a2a04916.

Reply:
Done in https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pr/pull/18/commits/8d7f2e455e8cf10fcb71ac6501ed5e72a2a04916.